commit d5c45e8051118ed76217e30ac45c48cd6fae395c
Author: Steven Newbury <steve@snewbury.org.uk>
Date:   Wed Dec 23 16:26:06 2015 +0000

    FindLLVM: check for empty system-libs variable
    
    When dynamic linking against LLVM, "llvm-config --system-libs" may be empty.
    Add check for a result before attempting to call REGEX REPLACE with an empty variable.
    
    Signed-off-by: Steven Newbury <steve@snewbury.org.uk>
    Reviewed-by: Yang Rong <rong.r.yang@intel.com>

diff --git a/CMake/FindLLVM.cmake b/CMake/FindLLVM.cmake
index a148321..32579f8 100644
--- a/CMake/FindLLVM.cmake
+++ b/CMake/FindLLVM.cmake
@@ -99,7 +99,9 @@ execute_process(
   OUTPUT_VARIABLE LLVM_SYSTEM_LIBS_ORIG
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
+if (LLVM_SYSTEM_LIBS_ORIG)
 string(REGEX REPLACE " *\n" "" LLVM_SYSTEM_LIBS ${LLVM_SYSTEM_LIBS_ORIG})
+endif (LLVM_SYSTEM_LIBS_ORIG)
 endif (LLVM_VERSION_NODOT VERSION_GREATER 34)
 
 macro(add_one_lib name)
